src/terms/entropy.jl

```diff
@@ -18,21 +18,23 @@ function ene_ops(term::TermEntropy, ψ, occ; kwargs...)
     T = eltype(basis)
     ops = [NoopOperator(term.basis, kpoint) for kpoint in basis.kpoints]
 
+    smearing = basis.model.smearing
     temperature = basis.model.temperature
+
     temperature == 0 && return (E=zero(T), ops=ops)
     ψ == nothing && return (E=T(Inf), ops=ops)
-    filled_occ = filled_occupation(basis.model)
 
+    filled_occ = filled_occupation(basis.model)
     eigenvalues = kwargs[:eigenvalues]
     εF = kwargs[:εF]
 
     E = zero(T)
     for (ik, k) in enumerate(basis.kpoints)
         for iband = 1:size(ψ[1], 2)
-            E -= temperature *
-                 basis.kweights[ik] *
-                 filled_occ * 
-                 Smearing.entropy(basis.model.smearing, (eigenvalues[ik][iband] - εF) / temperature)
+            E -= (temperature
```

Comment:
The reason I tend to prefer operators at the end of line is because it can lead to pretty subtle bugs otherwise:
```
x = 1
+ 2
```
gets you 1. In this case it's OK because of the parenthesis, but still. OTOH it's arguably more readable this way, so no strong opinion.

Reply:
Well you just need to get into the habit of putting parenthesis for mulit-line expressions.